.pre-commit-config.yaml

```diff
@@ -1,10 +1,10 @@
 repos:
 - repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: master
+  rev: main
   hooks:
     - id: trailing-whitespace
 - repo: https://github.com/python/black
-  rev: 21.6b0
+  rev: 22.3.0
   hooks:
     - id: black
```

Comment:
Maybe out of scope, but we usually set line-length to 120.

Reply:
i think the consensus on the team a long while back was to use black's defaults - but not sure